src/index.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable import/no-cycle */
 import './style.css';
-import moreIcon from './more.svg';
 import Icon from './enter.svg';
 import recycle from './recycle.svg';
+import { dragAndDrop } from './interactive.js';
 
```

Comment:
- [ ] You need to import here the new file (Status check)

Reply:
Done